src/layouts/Footer.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { makeStyles } from '@material-ui/core/styles';
+import Link from "@material-ui/core/Link";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles((theme) => ({
+  footer: {
+    justifyContent: "space-evenly",
+    alignItems: "center",
+    width: "100%",
+    height: "5vh",
+    position: "fixed",
+    bottom: 0,
+    left: 0,
+    backgroundColor: "#343B41",
```

Comment:
if you've got some design colors or fonts you should rewrite material defaults or just use defualts for example: `theme.pallete.primary.main`
https://material-ui.com/customization/palette/

same with fonts, you could use `theme.typography`
https://material-ui.com/customization/typography/

and spacing:
https://material-ui.com/customization/spacing/

Reply:
Added styling as discussed on Slack.